frontend/src/common/backend.ts

```diff
@@ -236,8 +235,7 @@ class VaultService {
 
   private async buildVaultAdminAuthorizationJWT(vaultId: string, vaultKeys: VaultKeys): Promise<string> {
     let vaultIdHeader: VaultIdHeader = { alg: 'ES384', b64: true, typ: 'JWT', vaultId: vaultId };
-    let nowInSeconds = this.secondsSinceEpoch();
-    let jwtPayload = { exp: nowInSeconds + vaultAdminAuthorizationJWTLeeway, nbf: nowInSeconds - vaultAdminAuthorizationJWTLeeway, iat: nowInSeconds };
+    let jwtPayload = { iat: this.secondsSinceEpoch() };
```

Comment:
What about the delta?

```js
let timeDiff = serverTime() - now(); // run this immediately after receiving the "config" response
// ... minutes later:
let iat = now() + config.getServerTimeDiff();
```

Reply:
Wanted to create a new PR with it, but here it is https://github.com/cryptomator/hub/pull/164/commits/d4cedb8dfcbb700c5bbc6d3a15a436b02f576d00

Reply:
instead of passing parameters around between vue components and backend.ts, can't the latter import `config` directly? It is a lazy singleton, after all.

Reply:
Correct :+1:, fixed in https://github.com/cryptomator/hub/pull/164/commits/6a0d3fedea5c73b8e75e887d03876c3d93483850